src/main/scala/uk/gov/hmrc/agents/accessgroups/UserDetails.scala

```diff
@@ -34,9 +28,5 @@ case class UserDetails(userId: Option[String] = None,
                        email: Option[String] = None)
 
 object UserDetails {
-
-  def fromAgentUser(agentUser: AgentUser): UserDetails =
```

Comment:
Do we have a plan to handle the sending the UserDetails to APFE from the backends?

Think this is currently used because we get a Set[AgentUser] and we want the emails for display. There is probably a better way that how we currently do it.

Reply:
I removed it from the library because it was very simple code and only used once. The same code is now inlined in the place where it is used.